pkg/daemon/ceph/osd/nsenter.go

```diff
@@ -72,10 +73,20 @@ func (ne *NSEnter) callNsEnter(binPath string) error {
 func (ne *NSEnter) checkIfBinaryExistsOnHost() error {
 	for _, path := range binPathsToCheck {
 		binPath := filepath.Join(path, ne.binary)
+		// Check with nsenter first
 		err := ne.callNsEnter(binPath)
 		if err != nil {
 			logger.Debugf("failed to call nsenter. %v", err)
-			continue
+			// If nsenter failed, let's try with the rootfs directly
+			rootFSBinPath := filepath.Join(rootFSPath, binPath)
```

Comment:
This will effectively check the following paths? It seems odd that the rootfs is mounted into all the pods, but ok.
- `/rootfs/usr/sbin/lvm`
- `/rootfs/sbin/lvm`

Reply:
The rootfs mount is for nsenter to do its job against the host mount namespace.